Makefile

```diff
@@ -380,6 +380,14 @@ cover: ## Runs the golang coverage tool. You must run the unit tests first.
 	$(GO) tool cover -html=cover.out
 	$(GO) tool cover -html=ecover.out
 
+test-data: start-docker ## Add test data to the local instance.
+	$(GO) run $(GOFLAGS) $(GO_LINKER_FLAGS) $(PLATFORM_FILES) sampledata -w 1
+
+	@echo You may need to restart the Mattermost server before using the following
+	@echo ==============================================================================
+	@echo Login with the system admin email=user-0@sample.mattermost.com password=user-0
```

Comment:
In the spinmints we have written a similar text, and we also included the normal account (user-1), I can imagine here another line like `@echo Login with a regular account with email=user-1@sample.mattermost.com password=user-1`